src/main/java/kappzzang/jeongsan/service/ExpenseService.java

```diff
@@ -56,9 +56,10 @@ public ExpenseResponse getResponses(Long memberId, Long teamId, Status status,
             filteredExpenses = new ArrayList<>();
         }
 
-        Integer totalPrice = expenses.stream()
+        Integer totalPrice = filteredExpenses.stream()
             .mapToInt(Expense::getTotalPrice)
-            .sum();
+            .reduce(Integer::sum)
+            .orElse(0);
```

Comment:
P3: 단순 덧셈이면 이렇게 쓸 수도 있는 것 같습니다 + sum도 빈 배열이면 0을 반환하지 않나요?
```suggestion
            .reduce(0, Integer::sum);
```

Reply:
그래도 명시적으로 나타내는 것이 코드를 이해하기 더 좋다고 생각했습니다